pages/_app.js

```diff
@@ -45,43 +45,56 @@ export default class HomePage extends App {
         </Head>
         <GlobalStyle />
         <header>
-          <div className="HeaderContainer">
-            <div className="CathoLogo">
-              <a href="/">
-                <img src="../static/imgs/cathoLogo.png" alt="Logotipo Catho" />
-              </a>
-            </div>
-
-            <nav id="visibleDesktopMenu" className="VisibleMenu">
-              <ul className="NavList">
-                <Hide small xsmall>
-                  <div className="VisibleMenu">
-                    <li>
-                      <a href="/cathocomponents">Catho Components</a>
-                    </li>
+          <Container withBreakpoints>
+            <Row>
+              <Col
+                medium={6}
+                small={4}
+                xsmall={2}
+                large={4}
+                className="CathoLogo"
+              >
+                <a href="/">
+                  <img
+                    src="../static/imgs/cathoLogo.png"
+                    alt="Logotipo Catho"
+                  />
+                </a>
+              </Col>
+              <Col medium={6} small={4} xsmall={2} large={8}>
+                <nav id="visibleDesktopMenu" className="VisibleMenu ">
+                  <ul className="NavList">
+                    <Hide small xsmall className="VisibleMenu">
+                      <li>
+                        <a href="/cathocomponents">Catho Components</a>
+                      </li>
+                      <li>
+                        <a href="/quantum">Quantum</a>
+                      </li>
+                    </Hide>
```

Comment:
Using Hide here generates a new "div" with "li" inside. Semantically it doesn't make sense to have a list (ul) with a div that contains several "li" inside. Hiding the list items using media query from a class makes more sense than using the Hide component. Another point, the use of the VisibleMenu class is not clear. For example: Without looking at the css, what does the VisibleMenu class do?? Show the menu? Why does the button element not have the class and Hide does? He understands?

If the Hide component is removed, the
VisibleMenu class will be used in only one scenario, that is, its use becomes more predictable, you know?